src/main/java/in/org/projecteka/hiu/consent/ConsentService.java

```diff
@@ -118,8 +119,8 @@ public ConsentService(HiuProperties hiuProperties,
                                                                  PatientConsentRequest consentRequest) {
         Map<String, String> response = new HashMap<>();
 
-        return Flux.fromIterable(consentRequest.getHipIds())
-                .flatMap(hipId -> validatePatientConsentRequest(requesterId, hipId)
+        return Flux.fromIterable(filterEmptyAndNullValues(consentRequest.getHipIds()))
```

Comment:
would hipIds ever be null or empty?

Reply:
It shouldn't be, but Qa tested the api with null and empty and they wanted to handle this scenario